Xamarin.Forms.Core/INavigationPageController.cs

```diff
@@ -7,7 +7,8 @@ namespace Xamarin.Forms
 {
 	public interface INavigationPageController
 	{
-		Stack<Page> StackCopy { get; }
+		Page SecondToLast { get;  }
```

Comment:
can't this be calculated from `IEnumerable<Page> Pages {get;}` ?

shouldn't it be internal ?